msg/en_US.utf8/cubrid.msg

```diff
@@ -1415,8 +1415,8 @@ Check the path of the key file (_keys) and make sure that it includes proper key
 1337 Log record does not exist (lsa : %1$lld|%2$d). The archive log volume has been deleted.
 1338 Flashback is already running. Flashback cannot be run by more than one user at the same time.
 
-1339 Unsupported argument type of stored procedure: %1$s
-1340 Unsupported return type of stored procedure: %1$s
+1339 (deleted)
```

Comment:
Trivial;
  Although the intention is slightly different, currently unused but reserved numbers are expressed in the following way.
"`Error reserved for...`"

Reply:
The intention seems slightly different. Mine was actually for  error numbers which are no more used and reusable by anyone,